lib/spack/docs/packaging_guide.rst

```diff
@@ -805,12 +788,12 @@ But its downloads are in many different subdirectories of
        homepage   = "http://www.mpich.org"
        url        = "http://www.mpich.org/static/downloads/3.0.4/mpich-3.0.4.tar.gz"
        list_url   = "http://www.mpich.org/static/downloads/"
-       list_depth = 2
+       list_depth = 1
 
 By default, Spack only looks at the top-level page available at
-``list_url``.  ``list_depth`` tells it to follow up to 2 levels of
-links from the top-level page.  Note that here, this implies two
-levels of subdirectories, as the ``mpich`` website is structured much
+``list_url``.  ``list_depth = 1`` tells it to follow up to 1 level of
+links from the top-level page.  Note that here, this implies 1
+level of subdirectories, as the ``mpich`` website is structured much
```

Comment:
Pretty sure this was a bug in `list_depth` that @tgamblin fixed a long time ago but the docs were never updated.